src/components/Home/About.tsx

```
@@ -89,6 +89,37 @@ export default function AboutSection() {
             *In a world of uncertainty, we provide clarity — empowering you to predict, protect, and prosper.*
           </p>
         </div>
+
+        {/* Structured Data for SEO */}
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{
+            __html: JSON.stringify({
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)